src/mpc-hc/mpcresources/PO/mpc-hc.de.strings.po

```diff
@@ -2767,7 +2770,7 @@ msgid ""
 "{\\an7\\1c&H000000&\\fs%d\\b0\\bord0\\shad0}File Name: %s\\N%sResolution: "
 "%dx%d %s\\NDuration: %02d:%02d:%02d"
 msgstr ""
-"{\\an7\\1c&H000000&\\fs%d\\b0\\bord0\\shad0}Dateiname: %s\\N%sAuflösung: "
+"{\\an7\\1c&H000000&\\fs16\\b0\\bord0\\shad0}Dateiname: %s\\N%sAuflösung: "
```

Comment:
This change is incorrect. I fixed it on Transifex yesterday. Transifex showed that this incorrect string was already a year old. So weird that it didn't show up in previous merge. Maybe also manual translation fixes in github source are not always synced to Transifex?

Reply:
Hmm.  https://github.com/clsid2/mpc-hc/commit/5ffdc07ac7387b046ab5d9e5ee6a59c9571a45d3 is the commit that overwrote it.  I think what happened is, this was sitting in transifex unmerged until the file hit 100%.  As a result the "old" translation showed up only now.  Let me check transifex for any other unmerged changes.

Reply:
Yeah I could see it on transifex, but couldn't find the commit.  I forced it to show back up now.